04_diy_neural/README.md

```diff
@@ -34,7 +34,19 @@
 * Video: [Data visualization for machine learning](https://vimeo.com/304131671)
 
 ### Assignment 4A due Wednesday, Sept 24, 9am
-* TBA (find and link to a dataset)
+1. Find a dataset that interests you. Some ideas:
+    * Something you find online. For example, take a look at [Kaggle](https://www.kaggle.com/) and you can find a list of datasets we've compiled on the [dataset wiki](https://github.com/ml5js/Intro-ML-Arts-IMA/wiki/Datasets).
+    * Find aa dataset that you collect yourself or is already being collected about you. For example, personal data like steps taken per day, browser history, minutes spent on your mobile device, sensor readings, and more.
+    * A dataset that you collect by crowdsourcing data. One way to approach this is to create a google form with questions and ask friends and fellow students to complete the form.
+2. After you have found or collected your dataset, write a blog post that addresses these questions:
+    * What is (are!) the data?
+    * What format is the data? (CSV, JSON, PDF, or . . )
```

Comment:
just a typo here I think - should be: What format is the data in?

Reply:
thank you! fixed with 2387e8fcaeade220faf15d3956d58fd9fbb39a4e